### ISSUE TITLE
right arrow shortcut in url bar

### ISSUE THREAD

**Issue by Gibbz** [QupZilla/qupzilla#2339]
Im used to firefox when typing in the url bar having the right arrow shortcut.
So for example i start typing slashdot. That then drops down from the auto search results. 
Then on firefix i would use the right arrow, which in qupzilla requires me first to press down, then enter instead.
I know its a small thing, but its pretty handy to have right arrow as select top item(down) and go(enter) in the single shortcut.

**Comment by nowrep** [comment]
I don't understand what you mean, can you please explain in more detail (maybe also add screenshots)?

**Comment by Gibbz** [comment]
So in qupzilla i press down then enter from the url bar to select the first
favourite in the list.
On firefox i can simply press right arrow to do the same thing.

Its pretty handy to have, and its pretty awkward not having that shortcut.

**Comment by nowrep** [comment]
Can you upload a video somewhere with this behavior? My Firefox behaves just like QupZilla (eg. pressing right arrow has no effect, I need to press down arrow to show completer popup)

**Comment by Vistaus** [comment]
@nowrep I think what he means is that Firefox fills the address bar with the first result, in this case being a bookmark. Just like when you (in Firefox and QupZilla) start to type a previously typed address.

@Gibbz You can easily workaround this by editing a bookmark in QupZilla and assigning a keyword. Then you can just type the keyword, in your case *slashdot* and you only have to press the Enter button then.

**Comment by Gibbz** [comment]
Yes, but having to assign a keyword for every bookmark is a pain.. that's
why the extra shortcut would be handy.

**Comment by Vistaus** [comment]
@Gibbz True, but it's still a workaround. Plus it's kind of browser-agnostic 'cause Firefox and SeaMonkey also support keywords for bookmarks. So you can easily go back and forth b/w QupZilla and FF/SeaMonkey keeping the same keywords.
